api-server/src/main/java/io/enmasse/api/v1/http/HttpAddressSpaceService.java

```diff
@@ -207,6 +206,10 @@ private AddressSpace setAddressSpaceDefaults(SecurityContext securityContext, St
             }
         } else {
             validateChanges(existing, addressSpace);
+
+            // Question - not sure of the intent here. This prevents the removal of any annotation (or label) which
```

Comment:
@lulf the existing code we have here prevents any label or annotation from being removed (either with patch or replace), even on that the user defines himself from his own domain.  This seems quite restrictive.   Addresses are similarly restrictive.

Reply:
Sorry @k-wall, resolved this by mistake. I agree we should relax the restriction to only cover our own annotations

Reply:
Sorry @k-wall, resolved this by mistake. I agree we should relax the restriction to only cover our own annotations